gremlin-go/driver/protocol.go

```diff
@@ -146,6 +146,7 @@ func (protocol *gremlinServerWSProtocol) close() (err error) {
 	protocol.mux.Lock()
 	if !protocol.closed {
 		err = protocol.transporter.Close()
```

Comment:
Nit: Remove named return value

Reply:
@simonz-bq This is outside of the scope of this ticket, would you mind creating a ticket to do this and placing it in our backlog? Thanks.